Assets/Scripts/Models/Furniture.cs

```diff
@@ -132,9 +132,9 @@ private Furniture(Furniture other)
         UnlocalizedDescription = other.UnlocalizedDescription;
     }
 
-    public event Action<Furniture> Changed;
+    public event Action<Furniture> OnChanged;
 
-    public event Action<Furniture> Removed;
+    public event Action<Furniture> OnRemoved;
```

Comment:
The bulk of events (13 out of 18) use On... name until and unless a different standard is discussed and agreed upon, I will be continuing to use this convention.